src/components/resultBox/resultBox.js

```diff
@@ -45,29 +45,17 @@ class ResultBox extends React.Component {
         </div>
 
         <footer>
-          <Button className={`okay-button ${styles.okButton}`}
-            onClick={() => {
-              this.props.transactionFailedClear();
-              // istanbul ignore else
-              if (typeof this.props.finalCallback === 'function') {
-                this.props.finalCallback();
-              }
-              this.props.reset();
-              this.props.history.replace(this.props.history.location.pathname);
-            }}>
-            {this.props.t('Okay')}
-          </Button>
           {this.props.success &&
-            this.props.reciepientId && this.isNotYetFollowed(this.props.reciepientId) ?
+            this.props.recipientId && this.isNotYetFollowed(this.props.recipientId) ?
             <Button className={`add-follwed-account-button ${styles.addFollowedAccountButton}`}
               onClick={() => {
                 this.props.nextStep({ address: this.props.reciepientId });
```

Comment:
The type needs to be fixed on this line too. Currently, the followed account is saved without the address.

```suggestion
                 this.props.nextStep({ address: this.props.recipientId });
```

Reply:
fixed